packages/schematics/README.md

```diff
@@ -0,0 +1,42 @@
+# Clarity Schematics
+
+This contains a schematics project for quickly scaffolding a new project with Clarity.
+
+### Installation
+
+```
+npm i -g @angular/cli
+```
+
+### Usage
+
+To create a new Clarity project, first create a new Angular project and then add Clarity.
+
+```
+ng new [project-name]
+cd ./project-name
+ng g add --collection=@clr/schematics
+```
+
+### Testing
+
+To test locally, install `@angular-devkit/schematics` globally and use the `schematics` command line tool. That tool acts the same as the `generate` command of the Angular CLI, but also has a debug mode.
+
+Check the documentation with
+```bash
+schematics --help
+```
+
+### Unit Testing
+
+`npm run test` will run the unit tests, using Jasmine as a runner and test framework.
+
+### Publishing
+
+To publish, simply do:
+
+```bash
+npm run build
+npm publish
```

Comment:
1. I don't see a publish script in the `packages/schematics/package.json` file. 
2. Is this how an update for our @clr/schematics is created and pushed to npm?

Reply:
`npm publish` is the normal publish command, so it will publish the files based on the package.json of the current directory. As long as you run the `build` command, it will compile the TypeScript and publish the resulting files. We can do something more automated in time, but its the minimal starting point.

Right now its not hooked into our release cycle, as I don't think we'll want to update this weekly.

Reply:
Done